Elizabeth_Theme/src/css/layout/_footer.scss

```diff
@@ -0,0 +1,41 @@
+#footer {
+    background-color: #000000;
+    color: #ffffff;
+
+    ul {
+      margin-left: auto;
+      margin-right: auto;
+
+      &.navbar-nav {
+        width: 410px;
+
+        .nav-item.hover:after {
+          width: auto;
+        }
+  
+        a {
+          color: #ffffff;
+          @include media-breakpoint-down(sm) {
+            padding-left: 6px;
+            padding-right: 6px;
+          }
+        }
+      }
+    }
+  
+    #socialMediaWrapper ul {
+      width: 192px;
+  
+      li a {
+        font-size: 2rem;
+      }
+    }
+  
+    p.powered-by a, .contact-info-container a {
+        color: #D6A261;
```

Comment:
Isn't this hex value color a variable in `_colors`?  Can we use the variables whenever needed?